common/locales/en/tasks.json

```diff
@@ -89,7 +89,7 @@
     "sureDelete": "Are you sure you want to delete this task?",
     "streakCoins": "Streak Bonus!",
     "pushTaskToTop": "Push task to top",
-    "pushTaskToBottom": "Push task to bottom",
+    "pushTaskToBottom": "Push task to bottom. Hold ctrl or cmd to push to top.",
```

Comment:
Actually, the pushTaskToTop isn't being used, I believe. I could be wrong though.